tests/unit/test_tfidf_utils.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+from reco_utils.recommender.tfidf.tfidf_utils import TfidfRecommender
```

Comment:
this guy is failing, transformers library needs to be aded to the generate_conda:
```
$ pytest tests/unit/test_tfidf_utils.py
========================================================================== test session starts ==========================================================================
platform linux -- Python 3.6.8, pytest-5.2.4, py-1.8.0, pluggy-0.12.0
rootdir: /home/miguel/repos/tests/recommenders
collected 0 items / 1 errors

================================================================================ ERRORS =================================================================================
____________________________________________________________ ERROR collecting tests/unit/test_tfidf_utils.py ____________________________________________________________
ImportError while importing test module '/home/miguel/repos/tests/recommenders/tests/unit/test_tfidf_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/unit/test_tfidf_utils.py:5: in <module>
    from reco_utils.recommender.tfidf.tfidf_utils import TfidfRecommender
reco_utils/recommender/tfidf/tfidf_utils.py:7: in <module>
    from transformers import BertTokenizer
E   ModuleNotFoundError: No module named 'transformers'
```
I would suggest to add the same version than the nlp repo: https://github.com/microsoft/nlp-recipes/blob/master/tools/generate_conda_file.py#L84

Reply:
same with nltk

Reply:
`./scripts/generate_conda_file.py` has been updated to included the necessary packages. I specified the same versions as in the nlp repo for transformers and nltk as suggested.